index.Rmd

```diff
@@ -32,32 +32,51 @@ After completing these modifications, your `.Rmd` should look like this [sample
 
 2.  Completely delete the YAML header (the section at the top of the `.Rmd` that includes name, title, date, output, etc.) including the `---` line.
     
-3. Choose a short, descriptive, human readable title for your project as your title will show up in the table of contents -- look at examples in the [rendered book](https://jtr13.github.io/cc20/){target="_blank"}. Capitalize the first letter only ("sentence case"). On the first line of your document, enter a single hashtag, followed by a single whitespace, and then your title. It is important to follow this format so that bookdown renders your title as a header. Do not use single # headers anywhere else in the document.
+3. Choose a short, descriptive, human readable title for your project as your title will show up in the table of contents -- look at examples in the [rendered book](https://jtr13.github.io/cc20/){target="_blank"}. Capitalize the first letter only ("sentence case"). On the first line of your document, enter a single hashtag, followed by a single whitespace, and then your title. It is important to follow this format so that bookdown renders your title as a header. Do not use single `#` headers anywhere else in the document. Note: if your chapter is the first in a section, a second `#` header and section description will be requested during the pull-request and merge phase.
     
 4. The second line should be blank, followed by your name(s):
     
-   ```
-    # Base R vs. ggplot2
+  ```
+  # Base R vs. ggplot2
     
-    Aaron Burr and Alexander Hamilton
-    Your content starts here. 
-   ```
+  Aaron Burr and Alexander Hamilton
+  Your content starts here. 
+  ```
 
 5. If your project requires data, please use a built-in dataset or read directly from a URL, such as:
 
     `df <- readr::read_csv("https://people.sc.fsu.edu/~jburkardt/data/csv/addresses.csv")` <br> If you absolutely must include a data file, please use a small one, as for many reasons it is desirable to keep the repository size as small as possible.
     
-6. If you have included a `setup` chunk in your `Rmd` file, please remember to remove the label `setup` in the chunk, ie., use :
+6. If you have included a `setup` chunk in your `Rmd` file, please remember to remove the label `setup` in the chunk, i.e., use:
```

Comment:
The spaces were intentionally so code indenting matches numbered list indenting (though I see it wasn't done consistently in 6.)

Reply:
It looked like they were intentional, but I didn't catch why, and chose to make the inconsistent indenting consistent everywhere. I'll update this, too.